Sources/Algorithms/Combinations.swift

```diff
@@ -153,9 +274,9 @@ extension Collection {
   /// That is, in the example above, the combinations in order are the elements
   /// at `[0, 1, 2]`, `[0, 1, 3]`, `[0, 2, 3]`, and finally `[1, 2, 3]`.
   ///
-  /// If `k` is zero, the resulting sequence has exactly one element, an empty
-  /// array. If `k` is greater than the number of elements in this sequence,
-  /// the resulting sequence has no elements.
+  /// If `kRange` is zero, the resulting sequence has exactly one element, an
+  /// empty array. If `kRange` is greater than the number of elements in this
+  /// sequence, the resulting sequence has no elements.
```

Comment:
I think these changes are the victim of an overaggressive find/replace 😄

Reply:
You’re right! It should be fixed now.